nservicebus/upgrades/7to8/index.md

```diff
@@ -177,4 +177,6 @@ endpointConfiguration.EnableOutbox();
 
 ### AbortReceiveOperation
 
-`ITransportReceiveContext.AbortReceiveOperation` operation is no longer supported
+`ITransportReceiveContext.AbortReceiveOperation` has been deprecated in favor of throwing an [`OperationCancelledException`](https://docs.microsoft.com/en-us/dotnet/api/system.operationcanceledexception).
```

Comment:
What is the difference with just throwing any exception?

Reply:
Good point, throwing any exception would have the same effect. @adamralph @DavidBoike I have a faint memory that we handled an OCE different in the pump early on. Do U remember?

Reply:
I'm assuming (maybe false assumption) that when using `AbortReceiveOperation` this action would be 'silent'. Meaning, error is logging with stack trace. Potentially only a DEBUG of INFO log entry.

Reply:
That is correct, you get full recoverability now which we believe is safer since the old way could result in "silent" infinite loops

Reply:
@ramonsmits tweaked the wording, take another look